### PR TITLE
Support for ARM64

### PR DESCRIPTION
**Pull request by JunielKatarn**
- Adds the `ARM64` build platform for both Desktop and Universal.
- Bumps ChakraCore.Debugger dependency to `0.0.0.38`.
- Bumps OpenSSL dependency to `1.0.2-p.3`.
- Bumps react-native dependency version to `v0.59.0-microsoft.20`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/2814)

### CODE REVIEW

File: .ado/publish.yml [hunk 1]
```diff
@@ -69,6 +69,9 @@ jobs:
         ArmDebug:
           BuildConfiguration: Debug
           BuildPlatform: arm
+        Arm64Debug:
```
Comment (JunielKatarn) [inline .ado/publish.yml]: @Khalef1 this file seems modified by your original branch.
Can your confirm these changes are intended?

File: .ado/templates/publish-build-artifacts-for-nuget.yml [hunk 2]
```diff
@@ -14,6 +14,20 @@ steps:
       arguments: '-TargetRoot $(Build.ArtifactStagingDirectory)'
     enabled: false
 
+  - task: PublishBuildArtifacts@1
```
Comment (JunielKatarn) [inline .ado/templates/publish-build-artifacts-for-nuget.yml]: @Khalef1 this file seems modified by your original branch.
Can your confirm these changes are intended?

**Comment by JunielKatarn** [comment]
Closing in favor of #3909

**Comment by akrantz** [comment]
@JunielKatarn Is this using ChakraCore-Debugger?

**Comment by JunielKatarn** [comment]
> @JunielKatarn Is this using ChakraCore-Debugger?

Not at this point.

**Pull request closed without merging**